src/commands/debugCommands.ts

```diff
@@ -0,0 +1,206 @@
+          this.terminal.log(
+            `${runModelProcess.formatText(runModelOutput.toString())}`,
+          );
+          if (runModelOutput.includes("ERROR")) {
+            throw new Error();
+          }
+        } catch (runError) {
+          console.log(runError);
+          window.showErrorMessage(
+            "Error running dbt debug for project " +
+              projectContext.label +
+              ". Please check the output tab for more details.",
+          );
+        }
+      } catch (err) {
+        console.log(err);
+        this.telemetry.sendTelemetryError("validateProjectError", err);
+      }
+    }
+  }
+  async installDeps(projectContext: ProjectQuickPickItem | undefined) {
+    if (projectContext === undefined) {
+      // This shouldnt happen really
+      window.showErrorMessage(
+        "No project was selected, please select a project in the step 'Pick a dbt project' above.",
+      );
+      return;
+    }
+    const answer = await window.showInformationMessage(
+      `Do you want to validate the project: ${projectContext.label}? This will run the command 'dbt deps' inside this project. Do you want to continue?`,
+      PromptAnswer.YES,
+      PromptAnswer.NO,
+    );
+    if (answer === PromptAnswer.YES) {
+      try {
+        this.telemetry.sendTelemetryEvent("installDeps");
+        const project = this.dbtProjectContainer.findDBTProject(
+          projectContext.uri,
+        );
+        if (project === undefined) {
+          window.showErrorMessage(
+            `Project ${projectContext.label} was not found`,
+          );
+          return;
+        }
+
+        const depsCommand = this.dbtCommandFactory.createInstallDepsCommand(
+          project.projectRoot,
+          project.dbtProfilesDir,
+        );
+        try {
+          const dbtDepsProcess =
+            await this.dbtProjectContainer.dbtClient?.executeCommand(
+              depsCommand,
+            );
+          const depsOut = await dbtDepsProcess.complete();
+          this.terminal.log(`${dbtDepsProcess.formatText(depsOut.toString())}`);
+        } catch (depsError) {
+          console.log(depsError);
+          window.showErrorMessage(
+            "Error installing dbt dependencies for project " +
+              projectContext.label +
+              ". Please check the output tab for more details.",
+          );
+        }
+      } catch (err) {
+        console.log(err);
+        this.telemetry.sendTelemetryError("installDepsError", err);
+      }
+    }
+  }
+
+  async isVsCodeOutdatated() {
+    this.telemetry.sendTelemetryEvent("vscodeOutdatedStep");
+    const currentVersion = version.toString();
+    const minVersion =
+      this.dbtProjectContainer.context?.extension.packageJSON.engines.vscode
+        .toString()
+        .trim()
+        .replace(/^[^<>~]/, "");
+    if (minVersion === undefined) {
+      return false;
+    }
+    return currentVersion < minVersion;
```

Comment:
This will never work as the installed vscode version will always be compatible with the installed extension.